relay-base-schema/src/metrics/mod.rs

```diff
@@ -25,7 +25,7 @@ pub fn try_normalize_metric_name(name: &str) -> Option<Cow<'_, str>> {
         return None;
     }
 
-    let normalize_re = NORMALIZE_RE.get_or_init(|| Regex::new("[^a-zA-Z0-9_/.]+").unwrap());
+    let normalize_re = NORMALIZE_RE.get_or_init(|| Regex::new("[^a-zA-Z0-9_.]+").unwrap());
```

Comment:
Why is `/` not replaced anymore?

Also, should the `.` be escaped? Or is it not interpreted as a wildcard when it occurs within `[...]`?

Reply:
> Why is / not replaced anymore?

👀 

> Or is it not interpreted as a wildcard when it occurs within [...]?

It's a literal `.` in a match group.

Reply:
`/` was exempt from replacing and is now replaced. The original spec allowed slashes, but for a long while SDK guidelines have excluded it. We're now aligning with the latest character set described in the epic and on develop docs.